src/components/molecules/Common/AssetModal/AssetListItem/index.tsx

```diff
@@ -10,11 +10,11 @@ import { metricsSizes } from "@reearth/theme/metrics";
 
 export type Asset = {
   id: string;
-  teamId: string;
+  teamId?: string;
   name: string;
-  size: number;
-  url: string;
-  contentType: string;
+  size?: number;
+  url?: string;
+  contentType?: string;
```

Comment:
Any reason to make teamId, size, url and contentType optional connected with google sheets? The Asset type in Golang and GraphQL expect all of these elements so we don't want to make them optional.

Reply:
In design I should use AssetList.
But google sheet API (for single sheets) does not provide any of these data (size, contentType) and also URL is not needed.

![2021-06-23_13-32](https://user-images.githubusercontent.com/78056580/123082257-7f54a980-d427-11eb-9941-6f8e34b14bbf.png)

Reply:
Ah I see, gotcha!